wiki/Tournaments/ATS/2018/en.md

```diff
@@ -0,0 +1,167 @@
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Userpage Banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Userpage Banner |
+
+*Rest prizes are to be decided*
+
+## Organization
+
+The Asian Taiko Showdown 2018 is run by various osu!taiko community member.
+
+| Position | Members |
+| ------------ | -------------- |
+| Organizer | [![][flag_ID] fajar13k](https://osu.ppy.sh/users/7100002), [![][flag_US] JDrago14](https://osu.ppy.sh/users/7690078) |
+| Map Selectors | [![][flag_ID] Guardistack-](https://osu.ppy.sh/users/1602428), [![][flag_MY] cdhsausageboy](https://osu.ppy.sh/users/2403621), [![][flag_ID] Lightning Wyvern](https://osu.ppy.sh/users/1533122)  |
+| Commentators | [![][flag_AU] Jaye](https://osu.ppy.sh/users/4841352), [![][flag_ID] Niko-nyan](https://osu.ppy.sh/users/906991) |
+| Streamers | *To be decided* |
+| Designers | [![][flag_PH] OsuMe65](https://osu.ppy.sh/users/852867), [![][flag_ID] fajar13k](https://osu.ppy.sh/users/7100002), [![][flag_CL] Valelup](https://osu.ppy.sh/users/5281416) |
+| Referees | [![][flag_HK] -GN Junior](https://osu.ppy.sh/users/3901754), [![][flag_CL] xfraczynho](https://osu.ppy.sh/users/2192669), [![][flag_CL] Valelup](https://osu.ppy.sh/users/5281416), [![][flag_CL] Asagaki](https://osu.ppy.sh/users/2632902), [![][flag_PH] Pochacco](https://osu.ppy.sh/users/2927742), [![][flag_ID] Alwaysyukaz](https://osu.ppy.sh/users/4999506), [![][flag_KR] MetalStream](https://osu.ppy.sh/users/165027) |
+| Statisticians | [![][flag_ID] Aozora-](https://osu.ppy.sh/users/6918206), [![][flag_US] JDrago14](https://osu.ppy.sh/users/7690078) |
+
+## Links
+
+- [Livestream](https://www.twitch.tv/idtslive)
+- [Registration Form](https://goo.gl/forms/Z40RvcISgnMKNwGf2)
+
+-----------------------
+
+## Participants
+
+Players will be displayed after the registration phase is completed.
+
+-----------------------
+
+## Mappool
+
+Mappools will be revealed one week in advance of each stage.
+
+------------------------------------------------------------------------
+
+## Ruleset
+### Tournament Rules
+
+1. The Asian Taiko Showdown is a one-on-one tournament, played on the osu!taiko game mode.
+2. In order for your registration to count, You must meet following condition:
+    - Participant must fill out the provided form.
+    - Participant's Flag on their profile must be a country from **Asia Continent.**
+    - Maximum Performace Points (PP) allowed for participant to register is **8,500pp.**
+    - Participant's account must be **6 months old or above.**
+3. Map scoring is based on **Score V2.**
+4. Match schedule will be settled by the Tournament Management (see below).
+5. If no staff or referee is available, the match will be postponed.
+6. Failed plays will result on enemy's winning.
+7. If the beatmap ends in a draw, the game will be nullified and proceed to next pick by next player.
+8. The maximum tolerance to attend is **10 minutes.**
```

Comment:
> 8. The maximum tolerance to attend is **10 minutes.**

What do you mean? Is this like the wait time before the match starts or something?

Reply:
`8. If a player doesn't show up within **10 minutes** of the start time, the enemy team wins by default.`

mostly from Sinnoh, just putting it here I guess